services/ymax-planner/src/main.ts

```diff
@@ -218,6 +233,37 @@ export const main = async (
     makeNonce: () => new Date(now()).toISOString(),
   });
 
+  const fetchAccountInfo: FetchAccountInfo = async address => {
+    return cosmosRest.getAccountSequence('agoric', address);
+  };
+
+  const sequenceManager = await makeSequenceManager(
+    {
+      log: (...args) => console.log('[SequenceManager]:', ...args),
+    },
+    {
+      address: signingSmartWalletKit.address,
+      fetchAccountInfo,
+    },
+  );
+
+  const smartWalletWithSequence = makeSmartWalletWithSequence(
+    {
+      signingSmartWalletKit,
+      sequenceManager,
+      log: (...args) => console.log('[SmartWalletWithSequence]:', ...args),
+    },
+    { chainId: networkConfig.chainName },
+  );
+
+  // create a wrapper that uses SmartWalletWithSequence methods
+  const smartWalletKitWithSequence: SmartWalletKitWithSequence = {
+    ...signingSmartWalletKit,
+    // override the three main methods to use SmartWalletWithSequence
+    sendBridgeAction: smartWalletWithSequence.sendBridgeAction,
+    executeOffer: smartWalletWithSequence.executeOffer,
+  };
```

Comment:
This is a smell (and inaccurately refers to "**three** main methods"); see my suggestion about the function in smart-wallet-with-sequence.ts returning an actual SigningSmartWallet rather than its component functions.